.github/workflows/check-pr-samples-3d-platformer.yaml

```diff
@@ -0,0 +1,40 @@
+name: Check PR - samples-3d-platformer
+on:
+  pull_request:
+    paths:
+      - 'entry-generation/godot-compiler-native-plugin/**'
+      - 'entry-generation/godot-compiler-plugin-common/**'
+      - 'entry-generation/godot-entry-generator/**'
+      - 'entry-generation/godot-annotation-processor/**'
+      - 'godot-kotlin/**'
+      - 'build.gradle.kts'
+      - 'buildSrc/**'
+      - 'gradle.properties'
+      - 'samples/3d-platformer/**'
+      - '.github/workflows/check-pr-samples-3d-platform.yaml'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            task: buildLinuxX64
+          - os: macos-latest
+            task: -x linkDebugSharedLinuxX64 buildMacosX64
+          - os: windows-latest
+            task: -x linkDebugSharedLinuxX64 buildWindowsX64
```

Comment:
Why isd the link step for linux necessary for windows and macOS?

Reply:
It's excluding the `linkDebugSharedLinuxX64 `, unfortunately `buildMacosX64` also triggers the link commands for linux. Also do note you can build linux binaries in macos and windows - if I don't exclude them the build lasts for 20mins+!

Reply:
ah sorry didn't pay attention to the `-x`